### PR TITLE
Fix Media Element Poster not showing on Windows/iOS/Mac

### LINKED ISSUE #1976: [BUG] No artwork shown in-app on MediaElement using meta data
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

Its not really a bug but also not really a new feature? 😅 

Using the new meta data properties on the `MediaElement` they show up wonderfully in the control centers and lock screens etc. but there is some inconsistency now with how its shown inside the app.

On Android, namely the artwork is shown on the `MediaElement` instance when there is no video to show, for example when there is only audio. 

But on iOS, macOS and Windows, it will not show. For Windows you just see nothing and for iOS & macOS you see the QuickTime logo (which I personally really hate).

I have been poking into showing the artwork for iOS on the `MediaElement` itself, but its hard to get it 100% right. I have been able to add the image and center it, however, you can still see the QuickTime logo underneath it. So I tried adding something artistic, like adding a blurred version of that same image behind the actual image, but I can't get that to have the right dimensions while also keeping in mind the memory usage here. You can see my hacky code here: https://github.com/CommunityToolkit/Maui/tree/dev/jfversluis/metadata-mediaelement

On Windows we can fix it relatively easily by doing: `mediaPlayerElement.PosterSource = new BitmapImage(new Uri(mediaElement.MetadataArtworkUrl));`

### Expected Behavior

`MediaElement` shows the artwork on the actual control inside the app when there is no visual media to show and the artwork meta data value is set.

### Steps To Reproduce

1. Open Toolkit sample app code
2. Go to MediaElement sample page
3. Change source to be an audio only source
4. Run the app on Android
5. Observe the artwork being shown (on the `MediaElement` control)
6. Run the app on any other platform
7. Observe the artwork not being shown (on the `MediaElement` control)

### Link to public reproduction project repository

https://github.com/CommunityToolkit/Maui

### Environment

```markdown
- .NET MAUI CommunityToolkit MediaElement: 4.0.0
- OS: Not important
- .NET MAUI: Not important
```

### Anything else?

I also talk about this in [my video](https://youtu.be/_9b36i79Ss0) and you can see it in action.

### PR DESCRIPTION
**Pull request by ne0rrmatrix**
- Bug fix

 ### Description of Change ###
Fix Media Element Poster not showing on screen during playback if no video and player is playing audio.

 ### Linked Issues ###
 - Fixes #1976

 ### PR Checklist ###

 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [x] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pulls 

 ### Additional information ###
This is an update that addresses complaints that device behavior is not consistent and this will unify behavior across all supported platforms. A poster from artwork will display if audio only inside the player when media element is active.

### CODE REVIEW

File: src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs [hunk 1]
```diff
@@ -296,13 +296,53 @@ public void Dispose()
...
 
+	void SetPoster()
+	{
+		
+		if (PlayerItem is null || metaData is null)
+		{
+			return;
+		}
+		var tracks = PlayerItem.Asset.Tracks;
+		if (tracks.ToList().Exists(track => track.MediaType.Contains("vide")))
```
Comment (jfversluis) [inline src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs:315]: vide? Not video?

**Review by jfversluis — COMMENTED** [review]
Would you have some screenshots on what this looks like?

Does the QuickTime logo still show underneath it for iOS?

File: src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs (hunk 1 above)
Comment (ne0rrmatrix) [reply src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs]: Changed to using `var videoTrack = PlayerItem.Asset.TracksWithMediaType(AVMediaTypes.Video.GetConstant()).FirstOrDefault();`
https://github.com/CommunityToolkit/Maui/assets/4167863/fa6893cc-65ae-4b15-81af-82a0e0403bc5

**Comment by ne0rrmatrix** [comment]
> Would you have some screenshots on what this looks like?
> 
> Does the QuickTime logo still show underneath it for iOS?

It is just a layer added on top to cover the quicktime logo. As far as I know there is no method that directly supports what we are doing in iOS. So as to whether quicktime logo is beneath the image I would say yes with a certainty.

File: src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs (hunk 1 above)
Comment (ne0rrmatrix) [reply src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs]: Player controls are still on top and not blocked by the poster either

**Comment by ne0rrmatrix** [comment]
Updated video with full screen behavior shown.

https://github.com/CommunityToolkit/Maui/assets/4167863/c61d4c39-ba9e-448f-a15c-cad2b91691f5

**Review by pictos — APPROVED** [review]

**Comment by ne0rrmatrix** [comment]
@jfversluis How does this PR look now? It works and does what is required. As to whether the quicktime icon is visible you cannot see it. The artwork covers it up. I can neither verify it is not there or have an easy test for it.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**